org.mixedrealitytoolkit.core/Utilities/Extensions/InteractorHandednessExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine.XR;
+using UnityEngine.XR.Interaction.Toolkit.Interactors;
+
+namespace MixedReality.Toolkit
+{
+    /// <summary>
+    /// Extension methods that make working with the <see cref="InteractorHandedness"/> enum easier.
+    /// </summary>
+    public static class InteractorHandednessExtensions
+    {
+        /// <summary>
+        /// Gets the <see cref="XRNode"/> representing the specified <see cref="InteractorHandedness"/>. If the <see cref="InteractorHandedness"/>
+        /// is other than InteractorHandedness.Left or InteractorHandedness.Right then it defaults to InteractorHandedness.Right.
+        /// </summary>
+        /// <param name="hand">The <see cref="InteractorHandedness"/> value for
+        /// which the <see cref="XRNode"/> is requested.</param>
+        /// <returns>
+        /// <see cref="XRNode"/> representing the specified <see cref="InteractorHandedness"/> with InteractorHandedness.Right as default.
+        /// </returns>
+        public static XRNode ToXRNodeWithRightHandDefault(this InteractorHandedness hand)
```

Comment:
This function name is a bit too verbose.  If you're wanting to add clarity here, perhaps have a default parameter.  How about this?

```cs
public static ToXRNode(this InteractorHandedness hand, XRNode defaultValue = XRNode.RightHand)
{
    switch (hand)
    {
        case InteractorHandedness.Left:
            return XRNode.LeftHand;
        case InteractorHandedness.Right:
            return XRNode.RightHand;
        default:
            return defaultValue;
    }
}
```

Reply:
Love it, implemented in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/743/commits/8938a7354c8ae853ff3df4eed5bb4a5c94312e3e , resolving.